ui/selectors/selectors.test.js

```diff
@@ -1017,6 +1017,37 @@ describe('Selectors', () => {
             balance: '0x0',
           },
         },
+        permissionHistory: {
```

Comment:
Is there a basic test we can add here?

Reply:
The test for this selector was updated, I have added this key and the values in the result https://github.com/MetaMask/metamask-extension/pull/23006/files#r1502163485